### DISCUSSION TITLE
我的也突然变成免费版，我登录专业版，却说我的激活码出错。

### DISCUSSION THREAD

**Discussion by helaowan** [getlantern/lantern-forum-cn#691] [General]
![2](https://github.com/getlantern/lantern-forum-cn/assets/174680292/e91a5f15-3288-4956-bf6b-79f82c3ffebd)
[我的也突然变成免费版，我登录专业版，却说我的激活码出错。然后，我重新购买的蓝灯专业版，用的信箱还是helaowan@126.com，结果钱花出去了，信箱里却没有收到任何信息，也没有新的激活码。我花了两份钱，原来的激活码无效，新买的专业版却没给我发新的激活码，这是怎么回事？我的账户：helaowan@126.com,我2024年1月购买的订单号是：2024011322001360251401427905，我2024年7月新买的订单号：2024070322001360251459110697
![Uploading 1.jpg…]()

**Comment by MaxMKMax** [comment]
@helaowan 您好. 我们会帮您解决问题. 请您放心. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ

*备注: 如果各位看到这个链接, 请您依然开个帖子说明您的问题. 这样我们才能发现到您的问题帮您跟进. 不要只点连接提交问题而不开帖子. 帖子上不需要个人资料了. 邮箱和支付单据放在链接里就好了. 谢谢!
